.github/workflows/deploy.yml

```diff
@@ -8,37 +8,119 @@ on:
     branches: [ master ]
 
 jobs:
-  deploy:
-    name: Deploy Demo App
+  deploy_chime_alpha_demo:
+    name: Alpha - Deploy Meeting Demo And Meeting Readiness Checker
```

Comment:
Saw Trung's comment on keeping the name same so can we keep it like: `name: Alpha - Deploy Demo App`` similarly elsewhere.

Reply:
Yeah, technically we deploy two demos here - no harm in having a more detailed title right?